appveyor.yml

```diff
@@ -278,6 +278,7 @@ build_script:
       ) else (
       if %BUILD_SYSTEM%==autotools (
         bash.exe -e -l -c "cd /c/projects/curl && ./buildconf && ./configure %CONFIG_ARGS% && make && make examples && cd tests && make"
+        bash.exe -e -l -c "cd /c/projects/curl && ls -alh **/curl.exe && ls -alh **/libcurl*.dll"
```

Comment:
it treats newlines as whitespace I think so for example
```
foo
bar
```
becomes `foo bar`

Reply:
Thanks!